<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>821</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1153</width>
    <height>821</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="btn_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>640</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Equalizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>451</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>520</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Equalizar2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>90</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Brillo:</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_b">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>120</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>40</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Contraste:</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_c">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_h">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>270</y>
        <width>311</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_abrir">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Save">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>530</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Binarizar">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>210</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Binarizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>150</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ecualizar</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_e">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>180</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>80</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>491</width>
        <height>411</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>260</y>
        <width>361</width>
        <height>291</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="open_img_A">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagen A</string>
      </property>
     </widget>
     <widget class="QPushButton" name="open_img_B">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Imagen B</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_A">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>261</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_B">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>251</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_resultado">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>481</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>120</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>OPERACIONES</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_suma">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string> A+B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_resta">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string> A-B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_divicion">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>(A+B)/2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_por_a">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>a*A+(1-a)*B</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_a">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>311</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tarea 3</string>
     </attribute>
     <widget class="QLabel" name="lbl_img_m">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>231</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_AA">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>231</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_M">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir M</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_AA">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir AA</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_D">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_D">
      <property name="geometry">
       <rect>
        <x>318</x>
        <y>10</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcula D = M-A</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_U">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular U</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_U">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_F">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>50</y>
        <width>211</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_F">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir F</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_img_R">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>370</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_R">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
